DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs

```diff
@@ -161,8 +161,10 @@ public void RemoveSignoffRequests(int selfAssessmentId, int delegateUserId, int
 						  ON CA.SelfAssessmentID = SAS.SelfAssessmentID
                     INNER JOIN CompetencyGroups AS CG 
 					ON SAS.CompetencyGroupID = CG.ID AND SAS.SelfAssessmentID =@selfAssessmentId
-                    WHERE ((ca.DelegateUserID = @delegateUserId) AND (ca.SelfAssessmentID = @selfAssessmentId) AND (sasr.SelfAssessmentReview = 1) AND (CG.ID =@competencyGroupId) AND (casv.SignedOff =0))
-                        OR ((ca.DelegateUserID = @delegateUserId) AND (ca.SelfAssessmentID = @selfAssessmentId) AND (sasr.SelfAssessmentReview IS NULL) AND (CG.ID =@competencyGroupId) AND (casv.SignedOff =0))
+                    WHERE ((ca.DelegateUserID = @delegateUserId) AND (ca.SelfAssessmentID = @selfAssessmentId) AND (sasr.SelfAssessmentReview = 1) AND

```

Comment:
as per the comment on jira, "We should be removing open requests (from CandidateAssessmentSupervisorVerifications) where Verified date is null, it looks like we are removing requests where SignedOff is 0 instead". We should remove the 
"(casv.SignedOff =0)" clause from here.

Reply:
done